system/camerad/sensors/ar0231.cc

```diff
@@ -116,6 +116,9 @@ AR0231::AR0231() {
   min_ev = exposure_time_min * sensor_analog_gains[analog_gain_min_idx];
   max_ev = exposure_time_max * dc_gain_factor * sensor_analog_gains[analog_gain_max_idx];
   target_grey_factor = 1.0;
+  ae_areas.push_back({96, 250, 1734, 524}); // wide
```

Comment:
any reason that this needs to live in the sensor class? it's nice that these sensor classes don't know anything about road/driver/etc.